packages/ohm-js/extras/extractExamples.js

```diff
@@ -73,6 +73,17 @@ const semantics = grammars.OhmWithExamples.createSemantics().addOperation('hasEx
 });
 
 semantics.addOperation('examples', {
+  Document(includesIter, grammarIter)
+  {
+    includesIter.examples();    
```

Comment:
I don't think it makes sense for `include` lines to have embedded examples. So you can change this as follows:

```suggestion
```